apps/ui/src/lib/next-helpers.ts

```diff
@@ -91,7 +92,7 @@ export async function getMetadataFromStrapi({
       { omitAuthorization: true }
     )
 
-    const seo = res.data?.attributes.seo
+    const seo = (res.data as any)?.seo
```

Comment:
Not entirely sure this is good, as we'll lose the typing for the entire SEO object. Was it necessary to remove the Extract type?